python/glow/gwas/lin_reg.py

```diff
@@ -23,11 +23,13 @@ def linear_regression(genotype_df: DataFrame,
                       add_intercept: bool = True,
                       values_column: Union[str, Column] = 'values',
                       dt: type = np.float64,
-                      verbose_output: bool = False) -> DataFrame:
+                      verbose_output: bool = False,
+                      intersect_samples: bool = False,
+                      genotype_sample_ids: Optional[List[str]] = None) -> DataFrame:
     '''
     Uses linear regression to test for association between genotypes and one or more phenotypes.
     The implementation is a distributed version of the method used in regenie: 
-    https://www.nature.com/articles/s41588-021-00870-7
+    https://www.biorxiv.org/content/10.1101/2020.06.19.162354v2
```

Comment:
Please keep this link pointing to the published version of the article in nature genetics